code-challenges.js

```diff
@@ -31,24 +52,85 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+
+const boilPoint = (fahrenheit) => { 
+        if (fahrenheit < 212){
+    return (`${fahrenheit} is below boiling point`)
+        }
+        else if (fahrenheit === 212){
+            return (`${fahrenheit} is at boiling point`)
+        }
+        else {
+            return (`${fahrenheit} is above boiling point`)
+        }
+    }
+    console.log(boilPoint(temp1)) 
+    // // Output: 42 is below boiling point
+
+    console.log(boilPoint(temp2))
+    // // Output: 350 is above boiling point
+
+    console.log(boilPoint(temp3))
+    // //Output: 212 is at boiling point
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+//built in method .concat
+//input: new variable created
+//output: number indicating total length of new combined array 
+
+// Process
+// Create a new 3rd variable
+//this 3rd variable will be the outcome of the 2 original arrays merged into 1 array
+//use an accessor (.concat)to merge the arrays
+//set the new variable equal to the merged arrays
+//run code
+
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+const newPadres= (padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+console.log(newPadres.length);
+
```

Comment:
Great work!  You can also chain .length to the end of the concat statement and then just console.log the variable name 🎉